about-page.html

```diff
@@ -0,0 +1,236 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="//use.fontawesome.com/releases/v5.0.7/css/all.css"
+    />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="style-desktop.css" />
+    <link rel="icon" href="img/e3-logo.png" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700;800&display=swap"
+      rel="stylesheet"
+    />
+    <title>About Us!</title>
+  </head>
+  <body>
+    <header>
+      <section class="menu-container">
+        <section class="mobile-menu dnone-mobile" id="m-menu">
+          <div class="menu-header">
+            <i class="fas fa-times" id="cancel-icon"></i>
+          </div>
+          <div class="menu-options">
+            <ul id="options-list">
+              <li><a href="index.html">Home</a></li>
+              <li><a href="about-page.html">About</a></li>
+              <li><a href="index.html#program">Program</a></li>
+              <li><a href="index.html#program">Join</a></li>
+              <li><a href="index.html#sponsor">Sponsor</a></li>
+              <li><a href="#past-events">Events</a></li>
+            </ul>
+          </div>
+        </section>
```

Comment:
- [ ] I think that it would have been better here if you had used a nav tag instead of a section tag.